Source/RestKit/RestRequest.swift

```diff
@@ -303,7 +303,36 @@ public struct RestRequest {
             completionHandler(dataResponse)
         }
     }
-    
+
+    public func responseVoid(
```

Comment:
No need to make a change right now, but at some point we should really go back to `RestKit` and comment a lot of these functions. That should make them easier to understand and maintain moving forward.